docs/data/data-grid/performance/GridWithReactMemo.js

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { unstable_useForkRef as useForkRef } from '@mui/utils';
+import {
+  DataGridPro,
+  GridRow,
+  DataGridProColumnHeaders,
+} from '@mui/x-data-grid-pro';
+import { useDemoData } from '@mui/x-data-grid-generator';
+
+const TraceUpdates = React.forwardRef((props, ref) => {
+  const { Component, ...other } = props;
+  const rootRef = React.useRef();
+  const handleRef = useForkRef(rootRef, ref);
+
+  React.useEffect(() => {
+    rootRef.current?.classList.add('updating');
+
+    const timer = setTimeout(() => {
+      rootRef.current?.classList.remove('updating');
+    }, 500);
+
+    return () => clearTimeout(timer);
+  });
+
+  return <Component ref={handleRef} {...other} />;
+});
+
+const RowWithTracer = React.forwardRef((props, ref) => {
+  return <TraceUpdates ref={ref} Component={GridRow} {...props} />;
+});
+
+const ColumnHeadersWithTracer = React.forwardRef((props, ref) => {
+  return <TraceUpdates ref={ref} Component={DataGridProColumnHeaders} {...props} />;
+});
+
+const MemoizedRow = React.memo(RowWithTracer);
+const MemoizedColumnHeaders = React.memo(ColumnHeadersWithTracer);
+
+export default function GridWithReactMemo() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 100,
+    editable: true,
+    maxColumns: 15,
+  });
+
+  return (
+    <Box
+      sx={{
+        height: 400,
+        width: '100%',
+        '& .updating': {
```

Comment:
```suggestion
        '&& .updating': {
```
To override the row hover background color, otherwise it might look like the row wasn't rerendered

Reply:
Looks like it's still not enough - `&&&` is needed to actually override the hover color